eng/scripts/Create-ApiReview.ps1

```diff
@@ -0,0 +1,185 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [array] $ArtifactList,
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,
+  [Parameter(Mandatory=$True)]
+  [string] $SourceBranch,
+  [Parameter(Mandatory=$True)]
+  [string] $DefaultBranch,
+  [Parameter(Mandatory=$True)]
+  [string] $ConfigFileDir,
+  [Parameter(Mandatory=$True)]
+  [string] $buildId,
+  [Parameter(Mandatory=$True)]
+  [string] $repoName,
+  [Parameter(Mandatory=$True)]
+  [string] $Language
+)
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-APIReview($packageArtifactname, $apiLabel, $releaseStatus, $reviewFileName)
+{
+    $params = "buildId=$buildId&artifactName=packages&originalFilePath=$packageArtifactname&reviewFilePath=$reviewFileName"
+    $params += "&label=$apiLabel&repoName=$repoName&packageName=$PackageName&project=internal"
+    $uri = "$($APIViewUri)?$params"
+    
+    Write-Host $uri
+    if ($releaseStatus -and ($releaseStatus -ne "Unreleased"))
+    {
+        $uri += "&compareAllRevisions=true"
+    }
+
+    $headers = @{
+        "ApiKey" = $APIKey;
+    }
+
+    try
+    {
+        $Response = Invoke-WebRequest -Method 'GET' -Uri $uri -Headers $headers
+        Write-Host "API Review URL: $($Response.Content)"
+        $StatusCode = $Response.StatusCode
+    }
+    catch
+    {
+        Write-Host "Exception details: $($_.Exception)"
+        $StatusCode = $_.Exception.Response.StatusCode
+    }
+
+    return $StatusCode
+}
+
+function ProcessPackage($PackageName)
+{
+    Write-Host "Artifact path: $($ArtifactPath)"
+    Write-Host "Package Name: $($PackageName)"
+    Write-Host "Source branch: $($SourceBranch)"
+    Write-Host "Config File directory: $($ConfigFileDir)"
+
+    $reviewFileName = "$($PackageName)_$($Language).json"
+
+    $packages = @{}
```

Comment:
Will there ever be more than one package?

Reply:
This function in existing script was written to be capable of processing multiple packages if individual language specific script sends more than one package info. I would like to retain this as is so we can migrate one by one language to send all package artifact details in one call to this script from language specific script returns artifact name.